LinqToRest/Async/IAsyncQueryProvider.cs

```diff
@@ -0,0 +1,31 @@
+﻿// from https://github.com/dotnet/efcore/blob/master/src/EFCore/Query/IAsyncQueryProvider.cs
+
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Linq;
+using System.Linq.Expressions;
+using System.Threading;
+using JetBrains.Annotations;
+
+namespace Messerli.LinqToRest.Async
+{
+    /// <summary>
+    ///     <para>
+    ///         Defines method to execute queries asynchronously that are described by an IQueryable object.
+    ///     </para>
+    ///     <para>
+    ///         The service lifetime is <see cref="ServiceLifetime.Scoped" />. This means that each
+    ///         <see cref="DbContext" /> instance will use its own instance of this service.
+    ///         The implementation may depend on other services registered with any lifetime.
+    ///         The implementation does not need to be thread-safe.
+    ///     </para>
```

Comment:
Remove this comment as it only applies to ASP.NET

Reply:
Fixed in bf28bde6d322597e5f66797777608ad663695518